docs/content/dev/getting-started.md

```diff
@@ -1,60 +1,138 @@
+7. Goto `hedgedoc/backend` directory with `cd ../backend`
+8. Create the `.env` config file by copying the example: `cp .env.example .env`
+9. Add a value to `HD_SESSION_SECRET` in the .env file. This can be any string, which has to be a secure password for production but can be set to simple string for debug purpose.
+10. Add the following lines
+   ```shell
+  echo "HD_AUTH_LOCAL_ENABLE_LOGIN=true" >> .env
+  echo "HD_AUTH_LOCAL_ENABLE_REGISTER=true" >> .env
+   ```
+11. Start the backend using `yarn start:dev &`
+12. Go to  `hedgedoc/frontend` directory with `cd ../frontend`
+13. Start the frontend using `yarn run dev:with-local-backend`
+14. Go to `hedgedoc/dev-reverse-proxy` with `cd ../dev-reverse-proxy`
+15. Start Caddy using `./caddy run` (when you manually downloaded the binary) or `caddy run` (when you installed Caddy via a package manager)
+16. Use your browser to go to <http://localhost:8080>
+
+## Preconditions
+
+If you want to run HedgeDoc in dev mode some preconditions have to be met.
 
 1. Make sure that NodeJS is installed. You need at least Node 14 (we recommend Node 18).
-2. Make sure that [yarn](https://yarnpkg.com/) is installed.
+2. Make sure that [Yarn](https://yarnpkg.com/) is installed.
 3. Clone this repo (e.g. `git clone https://github.com/hedgedoc/hedgedoc.git hedgedoc`)
 4. Go into the cloned directory
 
-### Installing the dependencies
+## Installing the dependencies
 
-Because we use yarn workspaces, yarn collects the dependencies of all packages automatically in one central top-level node_modules folder.
+Because we use Yarn workspaces, Yarn collects the dependencies of all packages automatically in one central top-level
+`node_modules` folder.
 To install the dependencies execute `yarn install` at the top level of the cloned repository.
 Execute this command ONLY there. There is no need to execute the install-command for every package.
-It's important to use [yarn](https://yarnpkg.com/). We don't support npm or any other package manager and using anything else than yarn won't work. 
+It's important to use [Yarn](https://yarnpkg.com/). We don't support `npm` or any other package manager and using anything
+else than Yarn won't work.
 
-### Compile the commons package
-Some code is shared by backend and frontend. This code lives in the "common" package and needs to be compiled so frontend and backend can import it.
+## Build `commons` package
+
+Some code is shared by backend and frontend. This code lives in the "commons" package and needs to be built so
+frontend and backend can import it.
 This only needs to be done once, except if you've changed code in the commons package.
+
 1. Go into the `commons` directory.
 2. Execute `yarn build` to build the commons package.
 
-## Setting up the backend
+## Setting up the Backend
+
+The backend can be mocked instead. For mock backend see below in frontend.
+
+**Note:** The backend can be mocked instead of starting it for real. See the "Mocked backend" section below.
```

Comment:
This is double information. Why do we mention it here anyway? That's more of a special case of the frontend.